models/clm/tutorial/README.rst

```diff
@@ -0,0 +1,1567 @@
+custom initial conditions and observation sequence files for your own work.
+
+
+.. Important ::
+
+  We have provided tutorial instructions for the NCAR
+  supercomputer Cheyenne, however, if using your own machine you will need to 
+  customize the setup scripts in order to properly compile DART (see Step 4:
+  Compiling DART). These system-specific setup steps may take a good deal of 
+  effort, especially if you are unfamiliar with details such as compilers, MPI,
+  NetCDF libraries, batch submission systems etc. To perform this tutorial we 
+  assume the user is comfortable with LINUX operating systems as well as using 
+  text editors (e.g. vi, nedit, emacs) to edit the CLM5-DART setup scripts 
+  and namelist files etc.
+
+  Other required files to run the tutorial include the meteorology (Step 5), 
+  reference case (Step 6), and observation files (Step 7).  These are all readily available
+  if you are using Cheyenne.  If you are using your own machine you need 
+  use the following links to download these files directly:
+                      
+  1. `CAM6 Reanalysis Meteorology <https://rda.ucar.edu/datasets/ds345.0/>`__,
+  Year 2011, ensemble members 1-5 only: ``f.e21.FHIST_BGC.f09_025.CAM6assim.011.cpl_000{1-5}.ha2x3h.2011.nc`` 
+  
+  2. `Reference Case and Observations <https://www.image.ucar.edu/pub/DART/CESM/clmdart_tutorial/>`__ 
+   
+
+Step 1: Download CLM5
+---------------------
+
+CLM is continually being updated by the model developer and user community
+consisting of both NCAR and university scientists and researchers.
+In contrast, DART is maintained by a relatively small group that supports
+numerous earth system models (20+) including CLM. Therefore the DART team
+focuses on only supporting official released versions of CLM.  This documentation
+and scripting was tested using the CESM tag ``release-cesm2.2.0`` following
+the download instructions `here <https://github.com/ESCOMP/CESM>`__.
+
+Although the DART code may work with more recent versions of CESM (CLM) we recommend
+checking out ``release-cesm2.2.0``.
+
+  ::
+
+    git clone https://github.com/escomp/cesm.git cesm_dart
+    cd cesm_dart
+    git checkout release-cesm2.2.0
+    ./manage_externals/checkout_external
+
+
+Adding CLM5 SourceMods
+----------------------
+
+Some minor modifications have to be made to the CLM5 source code in order
+to be run with DART. Most importantly, these include skipping several
+balance checks in CLM5 for the time step immediately after the assimilation
+update step.  These sourcecode modifications are brought in 
+through the SourceMod mechanism in CLM where modifications overwrite
+the template sourcecode during the compilation step. The SourceMods
+are located as tar files `here. <http://www.image.ucar.edu/pub/DART/CESM>`__
+For this tutorial use the most recent tar file ``DART_SourceMods_cesm2_2_0_2021_07_02.tar``
+and untar it on your local machine.  
+
+::
+ 
+  tar -xvf DART_SourceMods_cesm2_2_0_2021_07_02.tar
```

Comment:
I'd add some instructions on how to download the data directly to the machine. For instance:
`wget https://www.image.ucar.edu/pub/DART/CESM/DART_SourceMods_cesm2_2_0_2021_07_02.tar` 

This avoids the need to download (perhaps to a different machine) then copying the file to the right place.